app.js

```diff
@@ -16,6 +16,22 @@ app.get('/api/v1/palettes', (request, response) => {
     });
 });
 
+app.post('/api/v1/palettes', (request, response) => {
+  const palette = request.body
```

Comment:
I would add a semi at the end of this line and a line break before the next line!